about.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About-Section</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<header class="header">
+  <div class="f-dark">
+    <ul class="f-ul">
+      <li><img src="images/facebook.png" alt="facebook" width="100" class="f-logo"></li>
+      <li><img src="images/twitter.png"  alt="twitter" width="100" class="f-logo"></li>
+      <li>English</li>
+      <li>My Page</li>
+      <li>Logout</li>
+    </ul>
+  </div>
+  <nav class="white">
+    <div class="logo">
+        <a href="index.html"><img src="/images/reading-quran.png" alt="opened-Quran" width="30" height="39"></a>
+        <a href="index.html" class="black"><div>
+          <h2>AL IKRAAM <br> Institute</h2>
+        </div></a>
+    </div>
+    <ul class="n-about">
+      <li><a href="about.html" class="primary">About</a></li>
+      <li>Program</li>
```

Comment:
Great job @AhmedNazirMusah 🤝

- [ ] [Optional] Kindly avoid code duplication as this is not regarded as a good practice. The same `<header>` tag exists in both the `index.html` file and the `about.html` file. You can have one `<header>` tag and reuse it whenever you wish.

Reply:
Thanks, I have looked at the Images and rectified them.